manufacturing/filter_holder/README.md

```diff
@@ -34,22 +44,26 @@ We may have parts to send you from our inventory.
 
 | Item | Quantity | Manufacturer  | Part #               | Price (USD)      |Sources     | Notes |
 | ---- |---------:| ------------- | -------------------- | ----------------:|:-----------|:------|
-| 1    |        1 | RespiraWorks  | Filter Outer Housing | $1 (material)    | [B][1rw]   | 3D printed from data in link |
-| 2    |        1 | RespiraWorks  | Filter Inner Housing | $1 (material)    | [B][2rw]   | 3D printed from data in link |
-| 3    |        1 | Roomba        | Filter               | $8/6pk           | [Z][3amzn] | removed from filter unit as delivered |
-| 4    |        1 | RespiraWorks  | Filter Inner Gasket  | $0.10 (material) | B          | make using item 6 below |
-| 5    |        1 | RespiraWorks  | Filter Outer Gasket  | $0.10 (material) | B          | make using item 6 below |
-| 6    | ~4 sq in | McMaster-Carr | Gasket Material      | $22.22/ 12x12" sheet | [C][9mcmc] | cut with jigs to make items 4 and 5 |
-| 7    |        2 | McMaster-Carr | 96016A556            | $5.50/10pk       | [C][7mcmc] | thumbscrews |
-| 8    |        2 | McMaster-Carr | 94180A333            | $16.69/100pk     | [C][8mcmc] | thumbscrews |
+| 1    |        2 | RespiraWorks  | Filter Outer Housing | $1 (material)    | [B][1rw]   | 3D printed from data in link |
```

Comment:
And I guess this should then say "patient outer housing"

Reply:
and maybe switch with Item 2 below.

Reply:
i have doubt with the names, the panel mounting part is called as outer and the one to which patient circuit is connected is called inner, can you give me clarity about that?..

Reply:
I think the better terminology should be:
"outer housing with patient tubing interface"
and 
"outer housing without tubing interface"